### DISCUSSION TITLE
surveymarks/hu1253/

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#24] [Announcements]
# surveymarks/hu1253/

http://localhost:4000/surveymarks/hu1253/

**Comment by jengalas** [comment]
Posted by Joe Sullivan
October 15, 2023 at 10:02 pm
_Imported from WordPress_

What is the capacity and height of this tank

Thanks

**Reply by jengalas** [reply]
Posted by Zhanna
October 20, 2023 at 5:22 pm
_Imported from WordPress_

No idea on the height – probably 150 feet or more. Towers of this style usually hold 1 million gallons.
